### PR TITLE
Disallow capped collection creation when disabled

### PR DESCRIPTION
**Pull request by AlekSi**
# Description

Plus a small refactoring.

## Readiness checklist

- [ ] I added/updated unit tests (and they pass).
- [x] I added/updated integration/compatibility tests (and they pass).
- [x] I added/updated comments and checked rendering.
- [x] I made spot refactorings.
- [ ] I updated user documentation.
- [x] I ran `task all`, and it passed.
- [x] I ensured that PR title is good enough for the changelog.
- [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Milestone (`Next`), Labels, Project and project's Sprint fields.
- [x] I marked all done items in this checklist.

### CODE REVIEW

File: internal/handlers/sqlite/msg_create.go [hunk 1]
```diff
@@ -82,16 +82,13 @@ func (h *Handler) MsgCreate(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
...
-	if h.EnableOplog && capped {
-		var size any
-
-		size, _ = document.Get("size")
-		if size == nil {
-			msg := "the 'size' field is required when 'capped' is true"
-			return nil, commonerrors.NewCommandErrorMsgWithArgument(commonerrors.ErrInvalidOptions, msg, "create")
+	if capped {
+		if !h.EnableOplog {
+			return nil, common.Unimplemented(document, "capped")
```
Comment (AlekSi) [inline internal/handlers/sqlite/msg_create.go:87]: That's the critical part

Comment (chilagrow) [reply internal/handlers/sqlite/msg_create.go:87]: I see, only allow capped for oplog enabled.

**Review by chilagrow — APPROVED** [review]
Thanks for this

**Review by noisersup — APPROVED** [review]
The Capped() function looks more clear, LGTM

**Pull request merged**